src/rpc/client.cpp

```diff
@@ -112,7 +112,8 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "estimaterawfee", 0, "nblocks" },
     { "estimaterawfee", 1, "threshold" },
     { "estimaterawfee", 2, "horizon" },
-    { "prioritisetransaction", 1, "fee_delta" },
+    { "prioritisetransaction", 1, "dummy" },
```

Comment:
I dont think we care for position-independant arguments? They're still super new and I would be highly surprised if anyone is prioritizing by priority in 0.14, where its highly unsupported and disabled by default?